cmd/server/server.go

```diff
@@ -287,6 +288,11 @@ func setupEvilGlobals(c *cli.Context, v store.Store, r remote.Remote) {
 	// Cloning
 	server.Config.Pipeline.DefaultCloneImage = c.String("default-clone-image")
 
+	// Execution
+	server.Config.Pipeline.DeleteMultipleRunsOnEvents = regexp.
```

Comment:
c.StringSlice ?

Reply:
`StringSlice` is a way to get multiple options / strings from urfave-cli. I adjusted the code a bit to use it.